src/state/_common/helpers.js

```diff
@@ -25,6 +41,60 @@ function getUniqueIndexes(indexes) {
 	}
 }
 
+/**
+ * @param state {Object}
+ * @param filter {Object}
+ * @param order {Array}
+ * @param indexesData {Array}
+ * @param changedOn {string}
+ */
+function getUpdatedIndexes(state, filter, order, indexesData, changedOn) {
```

Comment:
What's "indexesData"? If I'm reading it right, it's an update for one index?

Reply:
Yes

Reply:
Then could we name it so?

Reply:
I did it :) , but GitHub does not let me tell you.

Reply:
name - indexUpdate, + document purpose & special case for creation